tensorcircuit/mpscircuit.py

```diff
@@ -79,71 +88,120 @@ class MPSCircuit(AbstractCircuit):
 
     """
 
-    # TODO(@SUSYUSTC): Add 3-qubits gates support
+    # TODO(@SUSYUSTC): fix the jax backend performance issue
 
     is_mps = True
 
     def __init__(
         self,
         nqubits: int,
+        center_position: Optional[int] = None,
         tensors: Optional[Sequence[Tensor]] = None,
-        center_position: int = 0,
+        wavefunction: Optional[Union[QuVector, Tensor]] = None,
```

Comment:
a good approach to combine `inputs` with `mps_inputs`, though the API deviates from `Circuit` class